### DISCUSSION TITLE
1.13.3report error

### DISCUSSION THREAD

**Discussion by SFheilian123** [Tyrrrz/YoutubeDownloader#526] [Q&A]
Hello author, the video on youtube can be watched normally, but app report video not availabie
![Snipaste_2024-10-04_22-06-09](https://github.com/user-attachments/assets/eedfc8ac-054a-441a-a942-d2da1dd373ff)

**Comment by Horiance** [comment]
same error

**Comment by teme2** [comment]
Hi,
Thank you for the SW but got same issue as above on the latest versions.
